sections/home/ImpactSection.jsx

```diff
@@ -77,3 +86,5 @@ const Title = styled(Text)`
 const TopWrapper = styled.div`
   padding: 0 5% 60px 5%;
 `;
+
+//TO DO - fix resizing image issue when the application window is resized smaller
```

Comment:
can you screenshot this error?

Reply:
don't worry this is good now I forgot to take this out, comment should be deleted now